src/libraries/Microsoft.Extensions.Hosting/src/HostAbortedException.cs

```diff
@@ -0,0 +1,54 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+using System;
+using System.Runtime.Serialization;
+
+namespace Microsoft.Extensions.Hosting
```

Comment:
Can we use file-scoped namespaces for new source code (like `StopTheHostException`)?
In my private projects I use them very much, but I don't know whats the rule for this.

Reply:
cc @stephentoub

Reply:
My preference would be "no", because then some files are different from others.

There are a couple of files, mostly involving non-public SafeHandle type decls in src/Common/src/Interop, that use multiple namespace scopes, and they can't be converted... so my preference is "never use the shortcutty thing"

Reply:
Okay. Then I'll leave the namespace style as it was in this PR. I will nevertheless start a small discussion for clarification in the next days generally to style changes :D.